Makefile

```diff
@@ -283,7 +290,9 @@ release: clean-release ## Builds and push container images using the latest git
 
 .PHONY: release-manifests
 release-manifests: $(RELEASE_DIR) ## Builds the manifests to publish with a release
-	$(KUSTOMIZE) build config/default > $(RELEASE_DIR)/cluster-api-components.yaml
+	$(KUSTOMIZE) build config/core > $(RELEASE_DIR)/cluster-api-components.yaml
```

Comment:
Output core yaml, kubeadm yaml, and "batteries included" yaml

Reply:
Sorry to sound a little pedantic, but I'd prefer to have something like

```
cluster-api.yaml (includes both)
core-components.yaml
bootstrap-components.yaml
```

The goal to bring kubeadm in tree was to make it seamless for new users to get up and running with cluster api.